Source/bmalloc/libpas/src/libpas/pas_try_allocate_common.h

```diff
@@ -165,9 +166,11 @@ pas_try_allocate_common_impl_slow(
             
             pas_physical_memory_transaction_begin(&transaction);
             pas_heap_lock_lock();
-            
-            result = pas_large_heap_try_allocate(
-                &heap->large_heap, size, alignment, config.config_ptr, &transaction);
+
+            if (config.pgm_enabled && pas_probabilistic_guard_malloc_should_call_pgm())
```

Comment:
Access `pas_pgm_can_use` directly here first to make the fast path fast (and avoid calling a function of `pas_probabilistic_guard_malloc_should_call_pgm`).